spec/controllers/api/v1/emission_targets_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe Api::V1::EmissionTargetsController, type: :controller do
```

Comment:
Don't know why, but unless I add `, format: :json`, all 3 controller specs fail, is it just me? Noted there's no travis configured, I'll get that sorted.

Reply:
:+1: